docs/release-notes/NuGet-6.2.md

```diff
@@ -0,0 +1,102 @@
+
+* Read and store the transitive origins of a package while reading installed packages from assets file - [#10751](https://github.com/NuGet/Home/issues/10751)
+
+* Add caching of the transitive dependencies data pulled from the lockfile (assets file) - [#10752](https://github.com/NuGet/Home/issues/10752)
+
+* Add TFM for .NET nanoFramework - [#10800](https://github.com/NuGet/Home/issues/10800)
+
+* Surface the transitive packages and its transitive origins through the search layer - [#11486](https://github.com/NuGet/Home/issues/11486)
+
+* [Feature]: Require package source mapping when using CPM - [#11505](https://github.com/NuGet/Home/issues/11505)
+
+* [Feature]: Allow overriding a centrally defined package version - [#11516](https://github.com/NuGet/Home/issues/11516)
+
+* [Feature]: Add support for a dedicated environment variable providing the NuGetScratch path. - [#11671](https://github.com/NuGet/Home/issues/11671)
+
+### Issues fixed in this release
+
+**DCRs:**
+
+* Make LocalPackageFileCache methods virtual - [#10325](https://github.com/NuGet/Home/issues/10325)
+
+* NuGetScratch lock files are not cleaned up - [#10679](https://github.com/NuGet/Home/issues/10679)
+
+* AutoCompleteResourceV3 does not use the supplied logger - [#11272](https://github.com/NuGet/Home/issues/11272)
+
+* [DCR]: Mitigate missing nuget.org when non-NuGet tool creates nuget.config without any sources - [#11387](https://github.com/NuGet/Home/issues/11387)
+
+* Add Author to the tooltip for a package in the packages list of PM UI - [#11499](https://github.com/NuGet/Home/issues/11499)
+
+* Remove NU5049  - [#11598](https://github.com/NuGet/Home/issues/11598)
+
+**Bugs:**
+
+* NuGet.exe list from local packages folder does not work with the AllVersion flag - [#4537](https://github.com/NuGet/Home/issues/4537)
+
+* Project A referencing package B via AssetTargetFallback, doesn't use that same AssetTargetFallback to pull B's dependency package C - [#5957](https://github.com/NuGet/Home/issues/5957)
+
+* Errors due to missing/failing sources are inconsistently shown in solution explorer vs the error list  - [#7245](https://github.com/NuGet/Home/issues/7245)
+
+* Arrow keys in NuGet PM UI Sources editing doesn't change order of persistence - [#8315](https://github.com/NuGet/Home/issues/8315)
+
+* PackageReference ungracefully handles duplicate Runtime Identifiers in csproj PackageReference  - [#9290](https://github.com/NuGet/Home/issues/9290)
+
+* RestoreIgnoreFailedSources=true still gives warnings - [#9765](https://github.com/NuGet/Home/issues/9765)
+
+* Introduce a warning for null/empty version range (new or reuse NU1604) - [#9767](https://github.com/NuGet/Home/issues/9767)
+
+* NuGet restore in VS should report progress to allow the project-system to better control design time build scheduling.  - [#9782](https://github.com/NuGet/Home/issues/9782)
```

Comment:
This should be a feature: https://github.com/NuGet/Home/issues/9782. 

We should include in the features with the title `Add IVsNuGetProjectUpdateEvents in Visual Studio, reporting of restore changes for PackageReference based projects.`  https://docs.microsoft.com/en-us/nuget/visual-studio-extensibility/nuget-api-in-visual-studio#ivsnugetprojectupdateevents-interface

Reply:
Should the link to the documentation be added in the release notes?